README.md

```diff
@@ -83,18 +83,23 @@ Information about contributing to this project.
 
 ## Buildah and Podman relationship
 
-Buildah and Podman are two complementary Open-source projects that are available on
-most Linux platforms and both projects reside at [GitHub.com](https://github.com)
-with [Buildah](https://buildah.io) [(GitHub)](https://github.com/containers/buildah) and
-[Podman](https://podman.io) [(GitHub)](https://github.com/containers/libpod).  Both Buildah and Podman are
-command line tools that work on OCI images and containers.  The two projects
-differentiate in their specialization.
+Buildah and Podman are two complementary open-source projects that are
+available on most Linux platforms and both projects reside at
+[GitHub.com](https://github.com) with Buildah
+[here](https://github.com/containers/buildah) and Podman
+[here](https://github.com/containers/libpod).  Both, Buildah and Podman are
+command line tools that work on Open Container Initiative (OCI) images images
```

Comment:
Images images?

Reply:
Fixed! Sorry for the silly typo.

Reply:
Also fixed in Buildah now https://github.com/containers/buildah/pull/1367 (Cc @TomSweeneyRedHat)

Reply:
and sniff, I reviewed that line in buildah and podman and missed it in both places....  Thanks for the pickup @mheon.

Reply:
I think the 80 character line breaks make it hard to review as the diffs can be very chaotic. In a previous life, when writing mostly LaTeX, I tried to have one sentence per line which made it easier.

Reply:
Agree here - markdown will mangle it into a paragraph no matter how we format it.